### PR TITLE
feat(Onboarding Role Selection): users can select their roles and nav…

### PR DESCRIPTION
**Pull request by Fodilahy-mena**
Hello team, 

The selection for a role task is done. Note that there are some changes with some components for improving some styling. 
Please have a look and give me some feedback in order to improve it.

Refs: U1 #41k8ak

Thanks in advance!

### CODE REVIEW

File: src/components/RoleSelectOption/RoleSelectOption.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Story, Meta } from '@storybook/react';
 import { Option, OptionProps } from './RoleSelectOption';
-import userIcon from "../../stories/assets/user.svg";
-import briefcaseIcon from "../../stories/assets/briefcase.svg";
+// import userIcon from "../../stories/assets/user.svg";
+// import briefcaseIcon from "../../stories/assets/briefcase.svg";
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.stories.tsx:4]: remove if unused

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 2]
```diff
@@ -36,60 +36,63 @@ const optionWrapperStyles = css`
...
+export const Option: React.FC<OptionProps> = ({ label, text }) => {
   return (
     <RoleSelectWrapper>
       <div
-        className={
-          label === 'Buyer'
-            ? 'inner-select-wrapper-buyer'
-            : 'inner-select-wrapper-seller'
-        }>
+        className="inner-select-wrapper">
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx:43]: we are using styled components so we should not be assigning class names.
Create separate styled component for each div/class name

File: src/components/RoleSelectOption/RoleSelectOption.tsx [hunk 3]
```diff
@@ -36,124 +36,100 @@ const optionWrapperStyles = css`
   border-radius: 6px;
 `
 
-export const Option: React.FC<OptionProps> = ({ label, text, src, alt }) => {
+export const Option: React.FC<OptionProps> = ({ label, text }) => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOption/RoleSelectOption.tsx:39]: Rename to `RoleSelectOption` to match file name

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx [hunk 4]
```diff
@@ -33,31 +34,27 @@ export const RoleSelectOnboardOne: React.FC<Props> = ({ footerLinks }) => {
...
+						<p>Form field of account registration will be here ....</p>
+					:
+					<div>
 							<JoinUsHeader>Join us</JoinUsHeader>
 							<DescriptionOffer
 								text={"To begin this story, tell us what kind of account you'd be opening "}
 							/>
-							<div>
+							{roleOptions.map(role => (
+								<div key={role.id} onClick={() => setRole(role)}>
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx:47]: `onClick` should be passed as a prop to `RoleSelectOption`

File: src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx [hunk 5]
```diff
@@ -16,12 +16,13 @@ import { PageFooter, Props } from '../PageFooter/PageFooter'
 import { Header } from '../Header'
 import { loggedIn } from '../HeaderNavLink/HeaderNavLink'
 import { Option } from '../RoleSelectOption/RoleSelectOption'
-import userIcon from '../../stories/assets/user.svg'
-import briefcaseIcon from '../../stories/assets/briefcase.svg'
 import { Login } from '../Login/Login'
 import { LeftSide } from '../LeftSide/LeftSide'
 
 export const RoleSelectOnboardOne: React.FC<Props> = ({ footerLinks }) => {
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx:22]: RoleSelectOnboardOne  rename to just RoleSelectOnboard

File: src/Pages/OnboardingRoleSelection.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        { linkContent: 'careers', linkUrl: '#careers' },
+        { linkContent: 'pricing', linkUrl: '#pricing' },
+        { linkContent: 'FAQ', linkUrl: '#faq' },
+      ],
+      id: '5',
+    },
+]
+
+
+function OnboardingRoleSelection() {
```
Comment (andrzej-onja) [inline src/Pages/OnboardingRoleSelection.tsx:57]: Lets rename this to OnboardingPage.tsx as it will also serve for register form later

**Review by andrzej-onja — APPROVED** [review]

**Review by vahona — APPROVED** [review]

**Review by starjardin — APPROVED** [review]

**Pull request merged**